easier_net/easier_net.py

```diff
@@ -0,0 +1,186 @@
+from ensurepip import bootstrap
+# from imblearn import RandomUnderSampler
+import numpy as np
+
+import pickle
+import sier_net #technically should be from sier_net import SierNetEstimator
+
+class EasierNetEstimator():
+    """ This is the Ensembling by Averaging Sparse-Input Hierarchical Networks(EASIER-NET)"""
+    #i think my init needs a seed
+    def __init__(
+        self, #num inits >=5
+        # seed=12,
+        # data_file="_output/data.npz",
+        num_classes=0,
+        input_filter_layer=True,
+        n_layers=2,
+        n_hidden=10,
+        num_inits=5,       #change this to n_estimators
+        max_iters=40,
+        max_prox_iters=0,
+        bootstrap=False,
```

Comment:
i dont think this is used, so remove

Reply:
I think we need n_estimators for creating the size of the ensemble? Removing the other 3 arguments.